pkg/containerd/importer/importer.go

```diff
@@ -60,6 +60,22 @@ func isLayerTar(name string) bool {
 	return slashes == 2 && strings.HasSuffix(name, "/layer.tar")
 }
 
+// followSymlinkLayer returns actual layer name of the symlink layer.
+// It returns "deadbeeddeadbeef/layer.tar" if the name is like
```

Comment:
nit.. the comments sometimes say deadbeeddeadbeef and sometimes say deadbeefdeadbeef replacing these with foo would take less space and hold the same meaning.

0xDEADBEEF holds historical significance as a marker for freed memory in a heap... if you are pointing to deadbeaf it means you have a bad pointer.